src/components/TopBar/CallButton.vue

```diff
@@ -191,6 +192,16 @@ export default {
 	}
 }
 
+.success {
+	color: white;
+	background-color: #35CC00;
+	border: 1px solid #35CC00;
+	&:hover {
```

Comment:
Don’t forget the `&:focus`! ;)

Reply:
Should this be kept with something like `darken($color-success, 7%) !default` ?
Or drop it, when we move to `var(--color-success)` @jancborchardt